docs/reference/builder.md

```diff
@@ -100,7 +100,7 @@ be UPPERCASE in order to distinguish them from arguments more easily.
 
 Docker runs the instructions in a `Dockerfile` in order. **The
 first instruction must be \`FROM\`** in order to specify the [*Base
-Image*](/terms/image/#base-image) from which you are building.
+Image*](/reference/glossary/#base-image) from which you are building.
```

Comment:
@thaJeztah @dharmit is fixing the issue as stated. To fix the change, requires adding a new definition and then linking to it.  I think that should be done in another PR.  The new glossary term would need to go through tech review.